YoCode/CheckManager.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+
+namespace YoCode
+{
+    internal class CheckManager
+    {
+        private readonly IPathManager dir;
+        private readonly List<Thread> workThreads;
+        private readonly bool isJunior;
+
+        public CheckManager(IPathManager dir, List<Thread> workThreads, bool isJunior)
+        {
+            this.dir = dir;
+            this.workThreads = workThreads;
+            this.isJunior = isJunior;
+        }
+
+        public ProjectRunner PassGatewayChecks(ICollection<FeatureEvidence> evidenceList)
+        {
+            var fileCheck = new FileChangeFinder(dir.ModifiedTestDirPath);
+            if (fileCheck.FileChangeEvidence.Failed)
+            {
+                evidenceList.Add(fileCheck.FileChangeEvidence);
+                return null;
+            }
+
+            var projectBuilder = new ProjectBuilder(dir.ModifiedTestDirPath, new FeatureRunner());
+            if (projectBuilder.ProjectBuilderEvidence.Failed)
```

Comment:
.Failed is no longer present in FeatureEvidence. To pass if statement for both false and null, it should look for: "!projectBuilder.ProjectBuilderEvidence.FeatureImplemented==truel"

Reply:
It is. I added it back in.